server.go

```diff
@@ -1187,6 +1191,28 @@ func (ctx *RequestCtx) TimeoutErrorWithResponse(resp *Response) {
 	ctx.timeoutResponse = respCopy
 }
 
+// NextProto adds nph to be processed when key is negotiated when TLS conection is established.
+//
+// This function MUST NOT be executed by different goroutines.
+func (s *Server) NextProto(key string, nph ServeHandler) {
+	if s.nextProtos == nil {
+		s.nextProtos = make(map[string]ServeHandler)
+	}
+	if s.tlsConfig == nil {
+		s.tlsConfig = &tls.Config{}
```

Comment:
Please add `PreferServerCipherSuites: true,` here. Otherwise calling `NextProto` before calling `AppendCert` causes it to be `false`. Better yet, add a function that initializes `tlsConfig` and use that function in [`AppendCert`](https://github.com/valyala/fasthttp/blob/b8ce34c94a0b8c3ad3a83c7bab4197373172476d/server.go#L1337) and [`AppendCertEmbed`](https://github.com/valyala/fasthttp/blob/b8ce34c94a0b8c3ad3a83c7bab4197373172476d/server.go#L1361) as well so we always initialize it the same way and avoid possible future bugs where things get added in one place but not in another.